src/main/scala/com/ing/wbaa/rokku/proxy/provider/AuthorizationProviderRanger.scala

```diff
@@ -65,9 +65,9 @@ trait AuthorizationProviderRanger {
       )
 
       rangerRequest.setAction(request.accessType.auditAction)
-      // We're using the original client's IP address for verification in Ranger. Ranger seems to use the
-      // RemoteIPAddress variable for this. For the header IPs we use the ForwardedAddresses: this is not
-      // completely true, but it works fairly enough.
+      //clientIp is used in audit
+      rangerRequest.setClientIPAddress(request.userIps.toString.take(200))
```

Comment:
Because users can put anything in the ip headers I want to protect the ranger audit to not overflow the audit record. 
If the ip record is truncated you can still check the full information in application log.
For most cases the limit is ok only for special attack on rokku the limit will be used but it is better to truncate the log van kill audit.
I have changed the code to be more clear.